docs/code_tour.md

```diff
@@ -2,8 +2,37 @@
 This document is intended to provide a development guideline for new and existing contributors to the project.
 Keep in mind this application undergoes constant development so be sure to check back occasionally as best practices are sure to evolve.
 
-## Linting
-This project uses **ESLint** and **Sass Lint**. Both can be run with the script `yarn vet`.
+## Yarn Tasks 
+#### `yarn build`
+- Bundles SUI assets and places then in `../manageiq/public/ui/service/`.
```

Comment:
![screenshot from 2017-04-26 13-44-57](https://cloud.githubusercontent.com/assets/6640236/25448464/91cb4172-2a86-11e7-90cc-2df0771c416f.png)

seems to be ok but can add 🤘